packages/gatsby-plugin-sharp/src/scheduler.js

```diff
@@ -1,6 +1,6 @@
 const { v4: uuidv4 } = require(`uuid`)
 const path = require(`path`)
-const fs = require(`fs-extra`)
+const fs = require(`fs`)
```

Comment:
Please keep fs-extra as it has better windows support.
```suggestion
const fs = require(`fs-extra`)
```